examples/simple/src/posts/TagReferenceInput.tsx

```diff
@@ -28,10 +34,18 @@ const TagReferenceInput = ({ ...props }) => {
         change('tags', []);
     };
 
+    const handleNewItem = (item: string) => {
+        console.log({ item });
+        return new Promise((resolve, reject) => {});
+    };
+
     return (
         <div className={classes.input}>
             <ReferenceArrayInput {...props} filter={{ published: filter }}>
-                <AutocompleteArrayInput optionText="name.en" />
+                <AutocompleteArrayInput
+                    optionText="name.en"
+                    onNewItem={handleNewItem}
```

Comment:
this change seems unrelated to the current PR, am I right?

Reply:
Yes, I messed up my commits. Working on too many branches :joy: